packages/base/src/dialogs/symbology/vector_layer/VectorRendering.tsx

```diff
@@ -1,31 +1,103 @@
-import React, { useEffect, useState } from 'react';
+import { LayerType } from '@jupytergis/schema';
+import React, { useState } from 'react';
 
 import { useGetProperties } from '@/src/dialogs/symbology/hooks/useGetProperties';
 import { ISymbologyDialogProps } from '@/src/dialogs/symbology/symbologyDialog';
 import {
   getColorCodeFeatureAttributes,
   getNumericFeatureAttributes,
+  objectEntries,
 } from '@/src/tools';
 import Canonical from './types/Canonical';
 import Categorized from './types/Categorized';
 import Graduated from './types/Graduated';
 import Heatmap from './types/Heatmap';
 import SimpleSymbol from './types/SimpleSymbol';
 
+type RenderType =
+  | 'Single Symbol'
+  | 'Canonical'
+  | 'Graduated'
+  | 'Categorized'
+  | 'Heatmap';
```

Comment:
Extract to `types.ts`

Reply:
Can we do it similarly to the `layerType`?